tools/odbc/duckdb_odbc.cpp

```diff
@@ -149,9 +149,11 @@ void OdbcHandleStmt::FillIRD() {
 		duckdb::DescRecord new_record;
 		auto col_type = stmt->GetTypes()[col_idx];
 
-		new_record.sql_desc_base_column_name = stmt->GetNames()[col_idx];
-		new_record.sql_desc_name = new_record.sql_desc_base_column_name;
-		new_record.sql_desc_label = stmt->GetNames()[col_idx];
+		// TODO: Make more specific?
```

Comment:
How would that work? Our columns only have names. Or are `name` and `label` ODBC specific?

Reply:
I just want to look into what's possible in terms of specifying these, even if it's throwing a `SUCCESS_WITH_INFO`.